docs/contributing.md

```diff
@@ -72,6 +72,23 @@ Most of our style is pretty easy to pick up on, but right now it's not entirely
 * In general we don't wrap lines, they can be as long as needed. If you do choose to wrap lines please do not wrap any wider than 76 columns.
 * We use `#pragma once` at the start of header files rather than old-style include guards (`#ifndef THIS_FILE_H`, `#define THIS_FILE_H`, ..., `#endif`)
 
+Here is an example for easy reference: 
+
+```c
+enum foo_state {
+  FOO_BAR,
+  FOO_BAZ,
+};
+
+int foo(void) {
+  if (cond1) {
+    return FOO_BAR;
+  } else {
+    return -1;
+  }
+}
+```
+
```

Comment:
I'd move this example above the `* We use C style comments` line and put it as another indented bullet point in the list, since it specifically relates to 1TBS and not the other stuff that comes after (comments, line wrapping, include guards).

Reply:
I added some comments to the example, as I think that's a better idea, and makes it fit at the end better.

Reply:
Good idea. The capitalization in the comments is inconsistent atm, though.